erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md

```diff
@@ -0,0 +1,32 @@
+<!-- add-breadcrumbs -->
+# Importing e-Invoice from Supplier
+ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government. Using this you can import Supplier e-invoices into ERPNext. The supplier details like supplier names, address and purchase invoices will get created automatically in the system from the XML files.
+
+## Prerequisites
+- Default Stock UOM should be specified in the Stock Settings Doctype.
+- Enable Check Supplier Invoice Uniqueness in the Accounts Settings Doctype.
+- Create a Zip file with all your supplier invoice XML files.
+
+## Instructions
+
+### On your ERPNext site:
+
+1. Navigate to Import Supplier Invoice doctype and entry the Invoice Series, Company, Supplier Group, Tax Account, Item Code and Default Buying Price list.
+	<img class="screenshot" src="/docs/assets/img/regional/italy/import_einvoice.png">
```

Comment:
the top part needs cropping, browser is leaking into the screenshot

Reply:
Are you asking to crop the image?

Reply:
yes

Reply:
Ok revised screenshot is attached, you can check.